src/ec/packages/ec-component-text-input/text-input.html.twig

```diff
@@ -10,6 +10,7 @@
   - "name" (string) (default: '')
   - "type" (string) (default: 'text')
```

Comment:
i noticed that we set the placeholder variable here even if we don't use it anymore... ;)

Reply:
done, I'll update this for the other components too..